client/src/components/Home/HomeRewards/HomeRewards.tsx

```diff
@@ -0,0 +1,87 @@
+import cx from 'classnames';
+import React, { ReactNode } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useAccount } from 'wagmi';
+
+import useEpochPatronsAllEpochs from 'hooks/helpers/useEpochPatronsAllEpochs';
+import useGetValuesToDisplay from 'hooks/helpers/useGetValuesToDisplay';
+import useIndividualRewardAllEpochs from 'hooks/helpers/useIndividualRewardAllEpochs';
+import useUserAllocationsAllEpochs from 'hooks/helpers/useUserAllocationsAllEpochs';
+import useIndividualReward from 'hooks/queries/useIndividualReward';
+
+import styles from './HomeRewards.module.scss';
+
+const HomeRewards = (): ReactNode => {
+  const { t } = useTranslation('translation', { keyPrefix: 'components.home.homeRewards' });
+  const { address } = useAccount();
+  const { data: individualReward, isFetching: isFetchingIndividualReward } = useIndividualReward();
+  const { data: individualRewardAllEpochs, isFetching: isFetchingIndividualRewardAllEpochs } =
+    useIndividualRewardAllEpochs();
+  const { data: userAllocationsAllEpochs, isFetching: isFetchingUserAllAllocations } =
+    useUserAllocationsAllEpochs();
+  const { data: epochPatronsAllEpochs, isFetching: isFetchingEpochPatronsAllEpochs } =
+    useEpochPatronsAllEpochs();
+  const getValuesToDisplay = useGetValuesToDisplay();
+
+  // We count only rewards from epochs user did an action -- allocation or was a patron.
+  const totalRewardsUsed = individualRewardAllEpochs.reduce((acc, curr, currentIndex) => {
+    const hasUserAlreadyDoneAllocationInGivenEpoch =
+      userAllocationsAllEpochs[currentIndex]?.hasUserAlreadyDoneAllocation || false;
+    const wasPatronInGivenEpoch =
+      epochPatronsAllEpochs[currentIndex]?.includes(address as string) || false;
+    const wasBudgetEffective = hasUserAlreadyDoneAllocationInGivenEpoch || wasPatronInGivenEpoch;
+
+    return wasBudgetEffective ? acc + curr : acc;
+  }, BigInt(0));
```

Comment:
Is it clear whether this is meant to be "used" only? "Total rewards" copy doesn't have "used" in it, maybe rewards skipped by the users should count too?

![image](https://github.com/user-attachments/assets/d636f536-9fb1-46c5-aedf-41c49a287f6a)

Reply:
This is copy of current implementation. I will change the name to `totalRewards` for clarity.